code/game/objects/items/grenades/chem_grenade.dm

```diff
@@ -11,7 +11,7 @@
 	force = 2
 	var/stage = EMPTY
 	var/list/beakers = list()
-	var/list/allowed_containers = list(/obj/item/reagent_containers/glass/beaker, /obj/item/reagent_containers/glass/bottle)
+	var/list/allowed_containers = list(/obj/item/reagent_containers/glass/beaker/solids)
```

Comment:
You can leave bottles on the list, that's fine - they're 30u after all.

Reply:
Exist large bottles, that works like a large beaker nappist, that's why i removed it

Reply:
they're glass/beaker/waterbottle, not /glass/bottle/waterbottle ; this means that waterbottles are actually a child of beakers, not a child of bottles, and are therefore a type of beaker according to the code.